db/data/queries/orderItemInfo.sql

```diff
@@ -1,15 +1,14 @@
-USE `quickOrder`;
```

Comment:
i need this statement to execute .sql files from the command line like this:
`mysql -h localhost -u someuser -p < orderItemInfo.sql`
If you remove this line, then the **command will fail**, because the database won't. know which database you are working with